.yarn/versions/13db328f.yml

```diff
@@ -0,0 +1,2 @@
+declined:
+  - "@gravitywelluk/auth"
```

Comment:
declined? What does this mean?

Reply:
Oops - the CLI to define versions breaks in my terminal sometimes. Updating!